omod/src/main/webapp/pages/myaccount/changeSecretQuestion.gsp

```diff
@@ -35,6 +35,7 @@ ${ ui.includeFragment("uicommons", "validationMessages")}
             ]) }
             ${ ui.includeFragment("uicommons", "field/text", [
                     id: "secret-question",
+                    initialValue: "$secretQuestion",
```

Comment:
I this working? Shouldn't it be secretQuestion without the quotes and dollar sign?